homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -0,0 +1,56 @@
+"""Platform for Husqvarna Automower base entity."""
+
+import logging
+from typing import Any
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import AutomowerDataUpdateCoordinator
+from .const import DOMAIN, HUSQVARNA_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AutomowerEntity(CoordinatorEntity[AutomowerDataUpdateCoordinator]):
+    """Defining the Automower base Entity."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    def __init__(self, coordinator: AutomowerDataUpdateCoordinator, idx: int) -> None:
+        """Initialize AutomowerEntity."""
+        super().__init__(coordinator, context=idx)
+        self.idx = idx
+        self.mower = coordinator.session.data["data"][self.idx]
+        self.mower_id = self.mower["id"]
+        self.mower_name = self.mower_attributes["system"]["name"]
```

Comment:
`self.mower_name` isn't used anywhere outside this method. Just make it a local variable.